src/styles/icons.scss

```
@@ -0,0 +1,28 @@
+.icon {
+  display: block;
+  height: $icon-size;
+  width: $icon-size;
+  background-size: cover;
+  background-position: center;
+
+  @include hover(transform, scale(1.2));
+
+  &--phone {
+    background-image: url(../images/icons/Icon-Phone.svg);
+  }
+
+  &--phone:hover::before {
+    content: url(../images/phone-numbers-hover.svg);
```

```suggestion
    content: url(../images/phone-numbers-hover.svg);
    position: absolute;
    right: 0;
```

add these styles to fix the placement of this pseudo-element